modules/woo-commerce/qwc-admin.php

```diff
@@ -41,9 +41,11 @@ function qtranxf_wc_add_filters_admin() {
     foreach ( $email_common as $name => $priority ) {
         add_filter( $name, 'qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage', $priority );
     }
-    
+
     add_filter( 'woocommerce_attribute_taxonomies' , 'qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage' );
     add_filter( 'woocommerce_variation_option_name' , 'qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage' );
+    //TODO: check upstream in QTX core if a cleaner approach can be adopted
+    add_action( 'woocommerce_after_edit_attribute_fields', function(){echo "<script>jQuery(document).ready(function(){jQuery(\"td\").show()})</script>";});
```

Comment:
Mmm yes we'd need something better. But how is this supposed to work? How is that code executed in the browser? Why just selecting any `td` element? Hard to understand.

Reply:
This is injected in one specific page only (woocommerce attribure edit screen) which has just a few labels and tds containing one input each. For some reason tds are hidden (hence page is unusable) due to some qtx js from upstream. Once the page is loaded the injected code shows up the tds again.
Considering that it's a very specific issue and modifying general upstream js may cause issues somewhere else, this workaround should be enough for now. However finding the actual root cause in the qtx js part it's not exactly trivial, it would take some time while the feature above is broken,

Reply:
> For some reason tds are hidden (hence page is unusable) due to some qtx js from upstream

Let's look at this better. It seems we can do a much cleaner fix

Reply:
Ok, in the meanwhile I have filled in #1162 for tracking.
This PR can be merged as a temporary workaround to guarantee usability or we can just leave the thing on hold while a better solution comes out, your choice.

Reply:
See
https://github.com/qtranslate/qtranslate-xt/issues/1162#issuecomment-1120239358

So this hook is simply undoing what QTS is doing earlier.
We can delay the release 3.12.0 until this is clarified, would be nice to have since Slugs seems quite well integrated with WC otherwise.

Reply:
If the solution to handle the attributes is to hard to implement with slugs active, let's keep the fields hidden for now.
But the WC script to undo what QTS is doing (specifically for WC) is not what we'd want for sure.

Reply:
Ouch, I completely forgot we moved WC stuff into slugs module, good catch! Those js workarounds are evil...
Reverted previous commit and modified existing QTS workaround with [d3e7688](https://github.com/qtranslate/qtranslate-xt/pull/1161/commits/d3e7688769fd00ec839fd26e2f457f1cc4f4f122)